code/modules/mob/living/simple_animal/friendly/pet.dm

```diff
@@ -3,6 +3,7 @@
 	mob_size = MOB_SIZE_SMALL
 	mob_biotypes = MOB_ORGANIC|MOB_BEAST
 	blood_volume = BLOOD_VOLUME_NORMAL
+	status_flags = GODMODE|CANPUSH // REMEMBER THIS ONE, GAMERS
```

Comment:
lol

Reply:
hmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmm

oh god what happens if there's a catsplosion of godmode cats

Reply:
you're not supposed to powergame the fall festival ath